src/main/java/com/fcgl/Listing/Vendors/Currency.java

```diff
@@ -0,0 +1,19 @@
+package com.fcgl.Listing.Vendors;
+
+public enum Currency {
+  USD("USD");
```

Comment:
There are max 300 + other Crypto, either that or store it in db. The string is not the way to go because that can mean a zillion things

Reply:
Yeah not string, but I think it would be weird to have 300 enums. Maybe getting them at runtime from DB would be better for this. But this works for now. MVP :)